jaeger_client/thrift.py

```diff
@@ -90,6 +97,18 @@ def make_tag(key, value, max_length):
         )
 
 
+def _make_traceback_tag(key, value, max_length):
+    key = _to_string(key)
+    value = traceback.format_exc()
```

Comment:
This looks weird - you're ignoring passed `value` and using some global state?

Reply:
Good point, I will format the traceback object `value` instead.